testgrid/specs/customer-migration-specs.yaml

```diff
@@ -50,6 +50,86 @@
     # write to the volume
     create_random_file_and_upload_to_deployment "migration-test" "default" "./test.data" "/data/test.data"
 
+    # deploy sample app
+    install_and_customize_kurl_integration_test_application
+  postUpgradeScript: |
+    # source helper functions
+    source /opt/kurl-testgrid/testhelpers.sh
+
+    # verify data was migrated
+    download_file_from_deployment_and_compare "migration-test" "default" "./test.data" "/data/test.data"
+
+    # ensure we can redeploy sample app
+    check_and_customize_kurl_integration_test_application
+
+    # ensure weave was removed
+    if ls /etc/cni/net.d/*weave* >/dev/null 2>&1; then
+      echo "weave CNI still installed on host"
+      exit 1
+    fi
+
+    # ensure kubernetes was upgraded
+    if [[ ! "$(kubectl version --short | grep -i server | awk '{ print $3 }' | sed 's/^v*//')" =~ "1.27" ]]; then
+      echo "Kubernetes was not upgraded to 1.27.x"
+      exit 1
+    fi
```

Comment:
Did you rebase? These changes are already in main: https://github.com/replicatedhq/kURL/blob/de78ea8f224e16ff17f5f1d957023155c7d96037/testgrid/specs/customer-migration-specs.yaml#L133-L155

Reply:
this is git being silly - the postinstallscript is the same so the diff is overlapping